### ISSUE TITLE
Calamares won't install the system on existing brtfs partition (Bugzilla Bug 1155)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#1155]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 1155
Date: 2015-03-16 11:51:09 +0000
From: @kyndek
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   @tpgxyz

Duplicates:   #1116
Depends on:   #1116
Last updated: 2015-03-18 08:07:31 +0000

## Comment 8767
Date: 2015-03-16 11:51:09 +0000
From: @kyndek

I'm trying to install latest build 9873 on ssd disk. In calamares I choose existing /home partition (brtfs) without format. Checking partition with brtfsck gives an error and installation stopped. When I ran btrfsck manually, gaves me no errors.

Here's the log from console:
http://wklej.to/0xhvp

## Comment 8768
Date: 2015-03-16 13:18:28 +0000
From: @tpgxyz

can you please run in terminal:

pkexec calamares -d

and try to go through installation process. When it fails attach output from terminal here.

## Comment 8771
Date: 2015-03-16 13:45:56 +0000
From: @kyndek

Created attachment 730
log

> Attached file: file_1155.txt (text/plain, 11274 bytes)
> Description:   log

## Comment 8772
Date: 2015-03-16 13:52:42 +0000
From: @kyndek

I wondering why calamares wants to resize sdb5 partition - I'dont want change size of sdb5....

## Comment 8778
Date: 2015-03-16 15:07:07 +0000
From: @tpgxyz

Can you please follow instruction from here:

http://calamares.io/bugs/browse/CAL-208?focusedCommentId=10423&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-10423

It may not related to calamares, but to your disk partitioning.

## Comment 8781
Date: 2015-03-16 18:46:00 +0000
From: @kyndek

Comment on attachment 732
log1

TPG, thanks for the tips, there were errors on btrfs partition, gparted didn't check a partition too. I backuped home partition and formatted it. I resolved one more problem with wrong path to squashfs.img:

ln -s /run/os-prober/dev/sdc/ /media/OpenMandrivaLx_2015.0

But now I have a problem with grub and efi. Here is the output...

## Comment 8786
Date: 2015-03-16 21:20:57 +0000
From: @tpgxyz

(In reply to comment #6)

> TPG, thanks for the tips, there were errors on btrfs partition, gparted
> didn't check a partition too. I backuped home partition and formatted it.

So it was badly formatted partition :)

> ln -s /run/os-prober/dev/sdc/ /media/OpenMandrivaLx_2015.0

Why you did that ?

By default ISO should be mounted in your /media dir out of box. Isn't this working for you ?

## Comment 8802
Date: 2015-03-17 10:31:40 +0000
From: @kyndek

I did the installation of cooker at last. In calamares I've chosen entire sdb disk with default partitioning. It created a boot partition with fat32 fs, and  a second root part with ext4. Maybe we must improve calamares to guide manual partitioning process for UEFI users, because seems to be complicated now.
Of course the problem with wrong path still exist for me. Look at the console output:

19:19:37 [0]: Starting job "Uruchom skrypt mount/main.py" 
19:19:37 [0]: Starting job "Uruchom skrypt unpackfs/main.py" 
19:19:37 [0]: Installation failed: 
19:19:37 [0]: - message: "Bad source" 
19:19:37 [0]: - details: "source="/media/OpenMandrivaLx_2015.0/LiveOS/squashfs.img"" 
^[[A^[[B19:23:06 [8]: Shutting down Calamares... 

Related to bug #1116.

## Comment 8803
Date: 2015-03-17 10:38:40 +0000
From: @tpgxyz

(In reply to comment #8)
> It created a boot partition with fat32
> fs, and  a second root part with ext4. 

FAT32 partition is mandatory for UEFI systems

>Maybe we must improve calamares to
> guide manual partitioning process for UEFI users, because seems to be
> complicated now.

There is a commit from upstream which should improve this situation.
https://github.com/calamares/calamares/commit/6a52811c30ea2a0fc491541af5be64d4c3c5db94

With new iso this should be imported to our calamares.

> "source="/media/OpenMandrivaLx_2015.0/LiveOS/squashfs.img"" 
> ^[[A^[[B19:23:06 [8]: Shutting down Calamares... 
> 
> Related to bug #1116.

When you boot iso, does /media directory is empty  ?

Please run ls /media and if there are any subdirectories please show theirs structure.

ISO should be mounted by udisks, maybe in UEFI this does not work somehow.

## Comment 8806
Date: 2015-03-17 11:22:14 +0000
From: @kyndek

Yeap, /media directory is empty

## Comment 8807
Date: 2015-03-17 12:41:56 +0000
From: @tpgxyz

Can you please boot iso and attach output of journalctl -b ?

## Comment 8815
Date: 2015-03-17 19:42:03 +0000
From: @kyndek

Created attachment 737
journallog

Here you are...

> Attached file: journal.log (text/x-log, 157831 bytes)
> Description:   journallog

## Comment 8818
Date: 2015-03-17 20:34:38 +0000
From: @tpgxyz

(In reply to comment #12)
> Created attachment 737 [details]
> journallog
> 
> Here you are...

Are you booting from usb stick or cd disk ?

## Comment 8819
Date: 2015-03-17 21:02:16 +0000
From: @kyndek

Usb stick writes with dd command...

## Comment 8841
Date: 2015-03-18 08:04:33 +0000
From: @tpgxyz

(In reply to comment #14)
> Usb stick writes with dd command...

I found the culrpit
https://issues.openmandriva.org/show_bug.cgi?id=1116#c20

## Comment 8842
Date: 2015-03-18 08:07:31 +0000
From: @tpgxyz

Duplicate of bug #1116

*** This bug has been marked as a duplicate of bug #1116 ***
